Analyzers/src/HNL_LeptonCore.C

```diff
@@ -581,8 +615,8 @@ AnalyzerParameter HNL_LeptonCore::SetupHNLParameter(TString s_setup_version, TSt
 
     param.FakeRateMethod       = "Standard";
     param.FakeRateParam        = "PtCone";
-    param.k.Muon_FR            = "AwayJetPt40";
-    param.k.Electron_FR        = "AwayJetPt40";
+    param.k.Muon_FR            = "FR_cent";
+    param.k.Electron_FR        = "FR_cent";
```

Comment:
changed central HNTight V2 Fakes, will need to check CR but better use latest FRs